### PR TITLE
Error Fixed

### PR DESCRIPTION
**Pull request by MacNew**

### CODE REVIEW

File: lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt [hunk 1]
```diff
@@ -48,52 +47,40 @@ open class LazyBase : AppCompatActivity() {
     }
 
     fun <T> Call<T>.fetch(
+        context: Context,
         response: (response: T?) -> Unit,
         failure: ((t: Throwable) -> Unit)? = null,
         showDialog: Boolean = true
     ) {
-        var dialog = DataDialog(this@LazyBase)
+        var dialog = DataDialog(context)
```
Comment (wisdomrider) [inline lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt]: pass activity context here not context

File: lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt [hunk 2]
```diff
@@ -48,52 +47,40 @@ open class LazyBase : AppCompatActivity() {
...
-
+             //       lazy.responseDialog.onSucess(response.body(), dialog, response.code())
+                    response(response.body())
             }
 
         })
     }
 
-
-    fun logd(s: Any) {
```
Comment (wisdomrider) [inline lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt:79]: yo kina metaunu vako :D logging xito grna rkheko the

**Review by wisdomrider — COMMENTED** [review]
yo fix garnuta ani murge grnu

File: lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt (hunk 1 above)
Comment (MacNew) [reply lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt]: I think I have remove DataDialog so you need to review Again ?

**Pull request merged**